features/step_definitions/cluster_configuration/localstorage.rb

```diff
@@ -1,245 +1,95 @@
+      | ["spec"]["storageClassDevices"][0]["storageClassName"] | #{name}      |
+    })
+    raise "Create localvolume #{name} failied" unless @result[:success]
   end
 
-  step %Q{I obtain test data file "storage/localvolume/configmap.yaml"}
-  cfm = YAML.load(@result[:response])
-  filepath = @result[:abs_path]
-  cfm["data"]["storageClassMap"].gsub!(/local-slow\D+/, "")
-  cfm["data"]["storageClassMap"].gsub!(/local-fast/, "block-devices")
-  cfm["data"]["storageClassMap"].gsub!(/\/mnt\/local-storage\S+/, "#{path}/block-devices")
-  File.write(filepath, cfm.to_yaml)
-  res = admin.cli_exec(:create, n: namespace, f: filepath)
-  raise "error creating configmap for local raw block devices storage provisioner" unless res[:success]
-
-  step %Q/I switch to cluster admin pseudo user/
-  sa = service_account(serviceaccount)
-  @result =  sa.create(by: admin)
-  step %Q/the step should succeed/
-
-  _opts = {scc: "privileged", user_name: "system:serviceaccount:#{namespace}:#{serviceaccount}"}
-  add_command = :oadm_policy_add_scc_to_user
-  @result = admin.cli_exec(add_command, **_opts)
-  step %Q/the step should succeed/
-
-  step %Q/I run the :create admin command with:/, table(%{
-      | f | #{BushSlicer::HOME}/testdata/storage/localvolume/local-block-template.yaml |
-      | n | #{namespace}                                                                                                     |
+  # check ds status
+  step %Q/I store the nodes in the :nodes clipboard/
+  step %Q/#{cb.nodes.count} pods become ready with labels:/, table(%{
+    | app=local-volume-diskmaker-local-storage |
   })
-  step %Q/the step should succeed/
-
-  step 'admin ensures "local-block:provisioner-node-binding" clusterrolebinding is deleted'
-  step 'admin ensures "local-block:provisioner-pv-binding" clusterrolebinding is deleted'
-  step %Q/I run the :new_app admin command with:/, table(%{
-      | param    | CONFIGMAP=#{configmap}            |
-      | param    | SERVICE_ACCOUNT=#{serviceaccount} |
-      | param    | NAMESPACE=#{namespace}            |
-      | param    | PROVISIONER_IMAGE=#{image}        |
-      | template | #{template}                       |
-      | n        | #{namespace}                      |
+  step %Q/#{cb.nodes.count} pods become ready with labels:/, table(%{
+    | app=local-volume-provisioner-local-storage |
   })
-  step %Q/the step should succeed/
 
-  nodes = env.nodes.select { |n| n.schedulable? }
-  step %/#{nodes.size} pods become ready with labels:/, table(%{
-      | app=local-volume-provisioner|
-  })
+  unless storage_class("#{name}").exists?
+    raise "Create storageclass #{name} failed"
+  end
+end
 
-  pv_count = 0
-  BushSlicer::PersistentVolume.list(user: admin).each { |pv|
-    pv_count += 1 if pv.name.start_with?("local-pv-") && (pv.local_path&.start_with?("#{path}/block-devices"))
-  }
+Given /^some local storage PVs are created successfully$/ do
```

Comment:
@qinpingli can you pls elaborate why you named the step `some local...`?  As it stands, it's kind of vague as to what `some` means.

Reply:
This step is used to prepare PVs for local storage plugin testing. The PV number depends on the cluster schedulable workers account. In this step, one loop device is created on each schedulable work, so the local storage provisioner will create one PV for each loop device.
If we want to test the local storage provisioner, we need another step.

Reply:
@qinpingli I think we should rename the step... how about `Given local storage PVs are created successfully in schedulable workers`?  WDYT @akostadinov @liangxia

Reply:
Thanks, updated.